static/templates/admin_user_list.hbs

```diff
@@ -17,8 +17,10 @@
             {{t "Administrator" }}
             {{else if is_guest}}
             {{t "Guest" }}
-            {{else}}
-            {{t "Member" }}
+            {{else if is_full_member }}
+            {{t "Full member" }}
+            {{else }}
+            {{t "New member" }}
```

Comment:
We should make sure this page has a link to `/help/roles-and-permissions`, maybe just next to the "Users" label, with our usual settings /help link system.

(Can you do this in a prep commit?)

Reply:
ok sure

Reply:
also we need to distingush between the full and new member  in the docs `/help/roles-and-permissions` right?